package.json

```diff
@@ -14,8 +14,8 @@
     "start:main": "ts-node-dev --inspect=0.0.0.0:9229 --respawn --transpile-only --rs src/main",
     "start:worker": "ts-node-dev --inspect=0.0.0.0:9230 --respawn --transpile-only --rs src/worker",
     "start:production": "run-p start:main:production start:worker:production",
-    "start:main:production": "node .",
-    "start:worker:production": "node src/worker",
+    "start:main:production": "run-s db:migrate:prod && node .",
+    "start:worker:production": "run-s db:migrate:prod && node src/worker",
```

Comment:
Should we run it on both worker and main? Should we just leave it only for main?

Reply:
I think it should be done on both as we don't know which one will spin up first and should assume it could be either.  I did confirm that the migration only happens once even though called in multiple processes because of the `SequelizeMeta` table.

Reply:
Doesn't really matter if one of the nodes will spin up before the db migration executed. We should never have a code which will rely that the migration will happen on the deployment before service startup. That mean our code should work perfectly with old and with the new state of the database. Only on the next step after the migration is completed we deploy the code change (or turn on a feature flag)

Reply:
Well, yes, it does.  Your new code needs to know the state of the database - if your code depends on a new table or column, you need to know that it's going to be there before the code runs.  This is how we can achieve that consistently.  The old code shouldn't break on an addition.  Since we can't directly change the DB schema from our local computers, we need something that can run side by side with the code.  This is a industry standard of doing DB maintenance.